src/components/reservation/ReservationModalBody.tsx

```diff
@@ -177,19 +172,20 @@ export const ReservationModalBody = ({
       );
     }
 
-    if (reservablePidsFromAnotherLibrary?.length && patron) {
-      const { patronId, name, emailAddress, preferredPickupBranch } = patron;
+    if (
+      reservablePidsFromAnotherLibrary?.length &&
```

Comment:
I think we should make a function or a variable called something like: `materialIsReservableFromAnotherLibrary`to reflect the domain instead of referring to some inner values. Does it make sense?

Reply:
It comes from the `useReservableFromAnotherLibrary` hook. Should I rename it there?

Reply:
I think it makes sense to create the variable right after this:
```typescript
  const reservablePidsFromAnotherLibrary = useReservableFromAnotherLibrary(
    selectedManifestations
  );
const materialIsReservableFromAnotherLibrary = Boolean(reservablePidsFromAnotherLibrary.length);
```

Reply:
Fixet